Added access control info to the vcd vdc info <vdc_name> command. and vcd vapp info <vapp_name>

anusuyar opened the pull request:
```
Added access control info to the vcd vdc info <vdc_name> command. and vcd vapp info <vapp_name>

By default, the VDC will have the following control access settings (Case1)
<ControlAccessParams>
    <IsSharedToEveryone>true</IsSharedToEveryone>
    <EveryoneAccessLevel>ReadOnly</EveryoneAccessLevel>
</ControlAccessParams>

But we can update it to have user specific access control (Case2)
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ControlAccessParams>
   <IsSharedToEveryone>false</IsSharedToEveryone>
   <AccessSettings>
      <AccessSetting>
        <Subject
            type="application/vnd.vmware.admin.user+xml"
            href="https://bos1-vcd-sp-static-199-204.eng.vmware.com/api/admin/user/d69620d1-1be4-41bc-8db3-ef1006dac028"/>
         <AccessLevel>ReadOnly</AccessLevel>
      </AccessSetting>
    </AccessSettings>
</ControlAccessParams>

Command output for Case1, 
C:\sources\github\service_ecosystem\vcd-cli (development) 
(vcd-cli) λ vcd vdc info vdc2(everyone_access_level and is_shared_to_everyone are the new access specific params)
property               value
---------------------  -------------------------------------------------------------------------------------------------------
everyone_access_level   ReadOnly
allocation_model       AllocationPool
cpu_capacity           {'units': 'MHz', 'allocated': '1980', 'limit': '1980', 'reserved': '990', 'used': '0', 'overhead': '0'}
id                     a9fa0a54-1c9d-44a6-81e3-4deaad599265
in_use                 False
is_enabled             True
is_shared_to_everyone  True
mem_capacity           {'units': 'MB', 'allocated': '1187', 'limit': '1187', 'reserved': '593', 'used': '0', 'overhead': '0'}
name                   vdc2
org                    AnuOrg
supported_hw           ['vmx-04', 'vmx-07', 'vmx-08', 'vmx-09', 'vmx-10', 'vmx-11']
vm_quota               100

Command output for Case2 (everyone_access_level param is absent in this case, but instead we have new per user access control settings.
C:\sources\github\service_ecosystem\vcd-cli (development) 
(vcd-cli) λ vcd vdc info vdc1
property               value
---------------------  --------------------------------------------------------------------------------------------------------
allocation_model       AllocationPool
cpu_capacity           {'units': 'MHz', 'allocated': '2200', 'limit': '2200', 'reserved': '1100', 'used': '0', 'overhead': '0'}
id                     eee20b45-8c2f-4b9a-bdd7-fc9cb224045a
in_use                 True
is_enabled             True
is_shared_to_everyone  False
mem_capacity           {'units': 'MB', 'allocated': '1310', 'limit': '1310', 'reserved': '655', 'used': '0', 'overhead': '0'}
name                   vdc1
org                    AnuOrg
supported_hw           ['vmx-04', 'vmx-07', 'vmx-08', 'vmx-09', 'vmx-10', 'vmx-11']
vm_quota               100

Access control settings:

access_level    user_href                                                                                              user_name
--------------  -----------------------------------------------------------------------------------------------------  -----------
ReadOnly        https://bos1-vcd-sp-static-199-204.eng.vmware.com/api/admin/user/d69620d1-1be4-41bc-8db3-ef1006dac028  user

Same applies for vapp, except EveryoneAccessLevel is absent in the XML.

```

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/anusuyar/vcd-cli/8)
<!-- Reviewable:end -->

Commit:
Changing the args format to pass param name and value.

Commit:
Merge branch 'master' into development

Commit:
Removed org_name in the update_org.
Since we are going to create a new method reload org. Not updating self.resource after updating
the org.

Commit:
passing the org_resource instead of org_href

Commit:
Merge branch 'master' into development

Commit:
Added access control info to the vcd vdc info <vdc_name> command.

By default, the VDC will have the following control access settings (Case1)
<ControlAccessParams>
    <IsSharedToEveryone>true</IsSharedToEveryone>
    <EveryoneAccessLevel>ReadOnly</EveryoneAccessLevel>
</ControlAccessParams>

But we can update it to have user specific access control (Case2)
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ControlAccessParams>
   <IsSharedToEveryone>false</IsSharedToEveryone>
   <AccessSettings>
      <AccessSetting>
        <Subject
            type="application/vnd.vmware.admin.user+xml"
            href="https://bos1-vcd-sp-static-199-204.eng.vmware.com/api/admin/user/d69620d1-1be4-41bc-8db3-ef1006dac028"/>
         <AccessLevel>ReadOnly</AccessLevel>
      </AccessSetting>
    </AccessSettings>
</ControlAccessParams>

Command output for Case1,
C:\sources\github\service_ecosystem\vcd-cli (development)
(vcd-cli) λ vcd vdc info vdc2(everyone_access_level and is_shared_to_everyone are the new access specific params)
property               value
---------------------  -------------------------------------------------------------------------------------------------------
everyone_access_level   ReadOnly
allocation_model       AllocationPool
cpu_capacity           {'units': 'MHz', 'allocated': '1980', 'limit': '1980', 'reserved': '990', 'used': '0', 'overhead': '0'}
id                     a9fa0a54-1c9d-44a6-81e3-4deaad599265
in_use                 False
is_enabled             True
is_shared_to_everyone  True
mem_capacity           {'units': 'MB', 'allocated': '1187', 'limit': '1187', 'reserved': '593', 'used': '0', 'overhead': '0'}
name                   vdc2
org                    AnuOrg
supported_hw           ['vmx-04', 'vmx-07', 'vmx-08', 'vmx-09', 'vmx-10', 'vmx-11']
vm_quota               100

Command output for Case2 (everyone_access_level param is absent in this case, but instead we have new per user access control settings.
C:\sources\github\service_ecosystem\vcd-cli (development)
(vcd-cli) λ vcd vdc info vdc1
property               value
---------------------  --------------------------------------------------------------------------------------------------------
allocation_model       AllocationPool
cpu_capacity           {'units': 'MHz', 'allocated': '2200', 'limit': '2200', 'reserved': '1100', 'used': '0', 'overhead': '0'}
id                     eee20b45-8c2f-4b9a-bdd7-fc9cb224045a
in_use                 True
is_enabled             True
is_shared_to_everyone  False
mem_capacity           {'units': 'MB', 'allocated': '1310', 'limit': '1310', 'reserved': '655', 'used': '0', 'overhead': '0'}
name                   vdc1
org                    AnuOrg
supported_hw           ['vmx-04', 'vmx-07', 'vmx-08', 'vmx-09', 'vmx-10', 'vmx-11']
vm_quota               100

Access control settings:

access_level    user_href                                                                                              user_name
--------------  -----------------------------------------------------------------------------------------------------  -----------
ReadOnly        https://bos1-vcd-sp-static-199-204.eng.vmware.com/api/admin/user/d69620d1-1be4-41bc-8db3-ef1006dac028  user

Commit:
Merge branch 'master' into development

Commit:
Retrieve vapp access control

sahithi commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[vcd_cli/catalog.py, line 143 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QlfbysHsT5IGN0YY5:-L0QlfbysHsT5IGN0YY6:b-b3jens) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/catalog.py#L143)):*
> ```Python
>         stdout('')
>         if access_settings is not None:
>             stdout(access_settings, ctx, show_id=True)
> ```

I suggest to print username instead of user_href. For other attributes like org, names are being printed.

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnQKjIIXcRNiLFRkb:-L0QnQKkyOUqGW_cRx8W:b-ysjz5z) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
> ```Python
>         if access_settings is not None:
>             stdout('\nAccess control settings:\n')
>             stdout(access_settings)
> ```

Same.. print username instead of href to keep it consistent with other attribute values.

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QpSF5NCiKy1VU5jG5:-L0QpSF5NCiKy1VU5jG6:b49rs9n) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
> ```Python
>         if access_settings is not None:
>             stdout('\nAccess control settings:\n')
>             stdout(access_settings)
> ```

Please pass ctx to stdout, It will fail to display in JSON if user chooses Json option.

---

*[vcd_cli/vdc.py, line 62 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QoQGTLJrqcg4XtCXB:-L0QoQGTLJrqcg4XtCXC:b-ej3t0v) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L62)):*
> ```Python
>         org_href = ctx.obj['profiles'].get('org_href')
>         org = Org(client, org_href)
>         vdc_resource = org.get_vdc(name)
> ```

Let the org.get_vdc() (Backend) throw the exception if vdc not found. IMO, we should try avoiding None checks on all entities in vcd-cli.
Please print vdc name in the error message.

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnD9Bh9lYyGMIMAfb:-L0QnD9CAoXbFXtBv2DB:b-jwnjjh) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
> ```Python
>         if access_settings is not None:
>             stdout('\nAccess control settings:\n')
>             stdout(access_settings)
> ```

Same here. I suggest to print username instead of href to keep it consistent with other attribute values.

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0Qpkn6TvT_kH9njfVv:-L0Qpkn6TvT_kH9njfVw:bbysxxp) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
> ```Python
>         if access_settings is not None:
>             stdout('\nAccess control settings:\n')
>             stdout(access_settings)
> ```

pass ctx to stdout

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8)*
<!-- Sent from Reviewable.io -->

pacogomez commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[vcd_cli/catalog.py, line 143 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QlfbysHsT5IGN0YY5:-L0R_UCaI1JyHUlXQfQs:b-a0f1xs) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/catalog.py#L143)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

I suggest to print username instead of user_href. For other attributes like org, names are being printed.
</blockquote></details>

this command might not be required if the same information goes in the `catalog info` command.

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QpSF5NCiKy1VU5jG5:-L0RaBA7T6YJDRYpaoDz:b-wtxo22) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Please pass ctx to stdout, It will fail to display in JSON if user chooses Json option.
</blockquote></details>

`access_settings should be merged in the same table as the one returned by `vapp_to_dict`. `vapp_to_dict` should have a third optional parameter `access_control=None` so the data can be merged in one single dict

---

*[vcd_cli/vdc.py, line 62 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QoQGTLJrqcg4XtCXB:-L0RazfELGBWARcyTWmE:b32v57) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L62)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Let the org.get_vdc() (Backend) throw the exception if vdc not found. IMO, we should try avoiding None checks on all entities in vcd-cli.
Please print vdc name in the error message.
</blockquote></details>

+1

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnD9Bh9lYyGMIMAfb:-L0Rb5Wm0gyG-EdVQJ4Q:bd4owhe) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Same here. I suggest to print username instead of href to keep it consistent with other attribute values.
</blockquote></details>

+1
Also see my comment about the vApp. Access settings should be merged in the same dict as the other properties.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8)*
<!-- Sent from Reviewable.io -->

Commit:
Addressing the review comment.
Displaying the acccess_control_settings as a flat list(as part of other info) instead of a table.

(vcd-cli) λ vcd vdc info vdc1
property                        value
------------------------------  --------------------------------------------------------------------------------------------------------
access-settings-1-access-level  ReadOnly
access-settings-1-subject-href  https://bos1-vcd-sp-static-199-204.eng.vmware.com/api/admin/user/d69620d1-1be4-41bc-8db3-ef1006dac028
access-settings-1-subject-name  user
access-settings-1-subject-type  application/vnd.vmware.admin.user+xml
allocation_model                AllocationPool
cpu_capacity                    {'units': 'MHz', 'allocated': '2200', 'limit': '2200', 'reserved': '1100', 'used': '0', 'overhead': '0'}
id                              eee20b45-8c2f-4b9a-bdd7-fc9cb224045a
in_use                          True
is_enabled                      True
is_shared_to_everyone           False
mem_capacity                    {'units': 'MB', 'allocated': '1310', 'limit': '1310', 'reserved': '655', 'used': '0', 'overhead': '0'}
name                            vdc1
org                             AnuOrg
supported_hw                    ['vmx-04', 'vmx-07', 'vmx-08', 'vmx-09', 'vmx-10', 'vmx-11']
vapp_templates                  []
vapps                           ['vapp2', 'vapp1']
vm_quota                        100

Also refactored get_catalog_access_settings to avoid duplication. and removed catalog control_access and move the info to catalog info command

anusuyar commented:
Review status: 0 of 3 files reviewed at latest revision, 6 unresolved discussions.

---

*[vcd_cli/catalog.py, line 143 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QlfbysHsT5IGN0YY5:-L0RXoGsSxsXk4VSoDn9:bo1ir03) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/catalog.py#L143)):*
<details><summary><i>Previously, pacogomez (Paco Gómez) wrote…</i></summary><blockquote>

this command might not be required if the same information goes in the `catalog info` command.
</blockquote></details>

I removed the control access command and moved the logic to info command
Paco and Aritra suggested to print all the info of a subject. Since a subject can be a user or a group, we need all details to make sense of the access_control_settings

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnQKjIIXcRNiLFRkb:-L0SAHvuUi2ICy4APixM:b-yl8vi6) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

Same.. print username instead of href to keep it consistent with other attribute values.
</blockquote></details>

Paco and Aritra suggested to print all the info of a subject. Since a subject can be a user or a group, we need all details to make sense of the access_control_settings

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QpSF5NCiKy1VU5jG5:-L0SAJJlOw9PlMIJ_Oqs:b-896fix) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
<details><summary><i>Previously, pacogomez (Paco Gómez) wrote…</i></summary><blockquote>

`access_settings should be merged in the same table as the one returned by `vapp_to_dict`. `vapp_to_dict` should have a third optional parameter `access_control=None` so the data can be merged in one single dict
</blockquote></details>

Done.

---

*[vcd_cli/vdc.py, line 62 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QoQGTLJrqcg4XtCXB:-L0SDlZYHoXa6mOA7azA:b-896fix) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L62)):*
<details><summary><i>Previously, pacogomez (Paco Gómez) wrote…</i></summary><blockquote>

+1
</blockquote></details>

Done.

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnD9Bh9lYyGMIMAfb:-L0SDnyUO3HgecNSMxrh:bvxtsvs) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
<details><summary><i>Previously, pacogomez (Paco Gómez) wrote…</i></summary><blockquote>

+1
Also see my comment about the vApp. Access settings should be merged in the same dict as the other properties.
</blockquote></details>

access setting merged to vdc_to_dict

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0Qpkn6TvT_kH9njfVv:-L0SHjlE93f_xWzpVJMU:b-qq42ym) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
<details><summary><i>Previously, sahithi (Sahithi Ayloo) wrote…</i></summary><blockquote>

pass ctx to stdout
</blockquote></details>

I removed that stdout. so not applicable now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8)*
<!-- Sent from Reviewable.io -->

rocknes commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-:-L0ST1bgkZ5c6uLk9FFB:bnfp4nl)*
<!-- Sent from Reviewable.io -->

pacogomez commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---

*[vcd_cli/catalog.py, line 143 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QlfbysHsT5IGN0YY5:-L0bdD4ZPGq8QeV_Le1J:bnfp4nl) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/catalog.py#L143)):*
<details><summary><i>Previously, anusuyar wrote…</i></summary><blockquote>

I removed the control access command and moved the logic to info command
Paco and Aritra suggested to print all the info of a subject. Since a subject can be a user or a group, we need all details to make sense of the access_control_settings
</blockquote></details>

<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnQKjIIXcRNiLFRkb:-L0bdKdIQulXOsODtj3G:bg7igkv) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
<details><summary><i>Previously, anusuyar wrote…</i></summary><blockquote>

Paco and Aritra suggested to print all the info of a subject. Since a subject can be a user or a group, we need all details to make sense of the access_control_settings
</blockquote></details>

<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

---

*[vcd_cli/vapp.py, line 119 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QpSF5NCiKy1VU5jG5:-L0bdModFJ3DxacEMiaF:bnfp4nl) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vapp.py#L119)):*
<details><summary><i>Previously, anusuyar wrote…</i></summary><blockquote>

Done.
</blockquote></details>

<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

*[vcd_cli/vdc.py, line 62 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QoQGTLJrqcg4XtCXB:-L0bdQDbZFtCkprhT8s3:bnfp4nl) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L62)):*
<details><summary><i>Previously, anusuyar wrote…</i></summary><blockquote>

Done.
</blockquote></details>

<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

*[vcd_cli/vdc.py, line 78 at r1](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-L0QnD9Bh9lYyGMIMAfb:-L0bdSScsuxDV1zeP_OL:bnfp4nl) ([raw file](https://github.com/anusuyar/vcd-cli/blob/7466027911f7735002af6a0ed470b90b156a270e/vcd_cli/vdc.py#L78)):*
<details><summary><i>Previously, anusuyar wrote…</i></summary><blockquote>

access setting merged to vdc_to_dict
</blockquote></details>

<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8#-:-L0bd_10Qz0SMtzp3-0S:bnfp4nl)*
<!-- Sent from Reviewable.io -->

sahithi commented:
Reviewed 3 of 3 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8)*
<!-- Sent from Reviewable.io -->

sahithi commented:
Reviewed 1 of 3 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/anusuyar/vcd-cli/8)*
<!-- Sent from Reviewable.io -->
